### ISSUE TITLE
Unexpected `raws.before` is added to new rules

### ISSUE THREAD

**Issue by HaoyCn** [postcss/postcss#1069]
See codes below:
```javascript
var postcss = require('postcss')
var plugin = postcss.plugin('postcss-test', function() {
	return function(css) {
		const clone = postcss.rule({ selector: '.b' })
		clone.append({ prop: 'height', value: '100px' })
		css.append(clone)
	}
})
postcss([ plugin ]).process('.a { *width: 100px }').then(function(css) {
	console.log(css.toString())
	/*
		.a { *width: 100px }
		.b { *height: 100px }
	*/
})
```
The bug is: the prefix `*` is attached to `height` property.

To avoid the behavior, I have to specify `raws.before` clearly:
```javascript
clone.append({
	raws: {
		before: css.first.first.raws.before.replace(/\S/g, '')
	},
	prop: 'height',
	value: '100px'
})
```

**Comment by ai** [comment]
Sure, it is bug. I will try to fix it on this weekend.

**Comment by ai** [comment]
Sorry, still did not find a time. Hope you have some hack fix :).

But I will fix it :).

**Comment by ai** [comment]
Fixed ceaf548

**Comment by ai** [comment]
Released in 6.0.12.
